integrations/pendle_lpt_integration.py

```diff
@@ -83,12 +83,12 @@ def get_participants(self) -> list:
         if self.participants is not None:
             return self.participants
 
-        logging.info(f"[{self.get_description()}] Getting participants...")
+        logging.info(f"[{self.integration_id.get_description()}] Getting participants...")
```

Comment:
running `python -m integrations.pendle_lpt_integration` failed as `get_description()` should be run on the `IntegrationId`.